tests/MercadoPago/Resources/Mocks/Response/Order/order_capture.json

```
@@ -0,0 +1,15 @@
+{
+  "id": "01HRYFWNYRE1MR1E60MW3X0T2P",
+  "status": "processed",
+  "transactions": {
+    "payments": [
+      {
+        "id": "01HRYFXQ53Q3JPEC48MYWMR0TE",
```

Peguei só mais um ajuste, o ID de um pagamento segue esse formato:
```suggestion
        "id": "pay_01JD2P9GGXAPBDGG6YT92J5305",
```